Core/Data.EF/Extensions/DatabaseSetupBuilderExtensions.cs

```diff
@@ -1,48 +1,49 @@
-﻿using Lens.Core.Data.EF.Services;
+﻿using Lens.Core.Data.EF.Repositories;
```

Comment:
I think the directives of the Repositories are not used in this file. You can remove them.